papermill/cli.py

```diff
@@ -61,6 +61,9 @@ def print_papermill_version(ctx, param, value):
 @click.option(
     '--parameters_base64', '-b', multiple=True, help='Base64 encoded YAML string as parameters.'
 )
+@click.option(
+    '--remove-tagged-cells', type=str, help='Remove cells with the specified tag before execution.'
```

Comment:
What about
```suggestion
    '--remove-cells-tagged', type=str, help='Remove cells with the specified tag before execution.'
```

Then it'd be pretty fluent on the command line:

```
--remove-cells-tagged=a-tag
```